doc/ansible/dev/advanced_options.md

```diff
@@ -50,3 +50,31 @@ You may want to manage what your operator watches and the owner references. This
 ```
 ENTRYPOINT ["/usr/local/bin/entrypoint", "--inject-owner-ref=false"]
 ```
+
+## Max Workers
+
+Increasing the number of workers allows events to be processed
+concurrently, which can improve reconciliation performance.
+
+Worker maximums can be set in two ways. Operator **authors** can set the
+default by including extra args to the operator container in
+`operator.yaml`. (Otherwise, the default is 1 worker.)
+
+``` yaml
+- name: operator
+  image: "quay.io/asmacdo/memcached-operator:v0.0.0"
+  imagePullPolicy: "Always"
+  args:
```

Comment:
Admins are actually able to set it here as well, as they will have control over the deployment. The only case this won't be able to be overridden is if the operator is deployed via OLM, in which case I believe that args is static.